### ISSUE TITLE
Client never sets `partialResultToken` and only sets `workDoneToken` during `initialize`

### ISSUE THREAD

**Issue by david-driscoll** [microsoft/vscode-languageserver-node#528]
Currently `workDoneToken` only gets set by the client during `initialize`.  And no place is the `partialResultToken` assigned as far as I can tell.

I'm not sure if it's implemented or not (leaning on not).

* Are these waiting for APIs on the vscode side to be completed before being plumbed through everything?
* How would these be implemented appropriately (to help by making a PR)?

**Comment by dbaeumer** [comment]
@david-driscoll VS Code has currently no support for work done and partial result progress. Although they are in the spec (mainly because other clients were asking for them) it will take a while until VS Code will support them. 

Is there a special feature you want to see supported first?

**Comment by david-driscoll** [comment]
Nope!  Usually these things land in VSCode around the same time as they do in the spec, as I was implementing them in the C# LSP library I wanted to validate that they were working as expected.

I was just want to make sure I wasn't missing something. 😄 Thanks!

**Comment by heejaechang** [comment]
@dbaeumer if we need to ping vscode people to add this new feature (incrementally update UI/results for Workspace Symbols or Find All References) who should we talk to?

**Comment by dbaeumer** [comment]
@heejaechang you need to open an issue against VS Code (see also https://github.com/microsoft/vscode/issues/20010)

**Comment by heejaechang** [comment]
opened https://github.com/microsoft/vscode/issues/105870
